test/WebSocketSession.hpp

```diff
@@ -247,7 +256,7 @@ class WebSocketSession final : public WebSocketHandler
     {
         {
             std::unique_lock<std::mutex> lock(_outMutex);
-            _outQueue.put(std::vector<char>(msg.data(), msg.data() + msg.size()));
+            _outQueue.emplace_back(std::vector<char>(msg.data(), msg.data() + msg.size()));
```

Comment:
nit: emplace_back doesn't need the intermediate temp so could be:
_outQueue.emplace_back(msg.data(), msg.data() + msg.size());